src/main/java/com/example/demo3/DbmsConnection.java

```diff
@@ -5,32 +5,25 @@
 import java.sql.SQLException;
 import java.sql.Statement;
 
-public final class dbmsConnection {
+public final class DbmsConnection {
     // JDBC URL, username and password of MySQL server
     private final String url;
     private final String username;
     private final String password;
 
-    private static dbmsConnection INSTANCE;
+    private static DbmsConnection INSTANCE;
 
-    private dbmsConnection() {
+    private DbmsConnection() {
         this.url = "jdbc:mysql://89.184.93.8:3306/javafx_aibot";
         this.username = "u_javafx_aib";
         this.password = "Ul1SwXimEQ9W";
 
         String info = "Initial info class";
         System.out.println(info);
     }
-    /*private dbmsConnection() {
```

Comment:
Keep clean your code, remove all commented code that not used from classes. Or keep it locally not in repo.